site/app/templates/admin/admin_gradeable/AdminGradeableCreate.twig

```diff
@@ -533,6 +533,26 @@
         //
         // Gradeable Type Radio Button Events
         //
+        
+        $("input[type='radio']").change(validate);
+        
+        function validate() 
+        {
+            var sub1 = $('input[type]');
+            var sub2 = $('input[electronic_gradeable_presets]');
```

Comment:
I think `const` would be better than `var` here since there is no reason for these variables to change under this function